src/goose/toolkit/developer.py

```diff
@@ -1,24 +1,26 @@
 import os
-import re
-import subprocess
-import time
 from pathlib import Path
 from typing import Dict, List
 
 from exchange import Message
 from goose.toolkit.base import Toolkit, tool
 from goose.toolkit.utils import get_language, render_template
-from goose.utils.ask import ask_an_ai
-from goose.utils.check_shell_command import is_dangerous_command
 from rich.markdown import Markdown
 from rich.prompt import Confirm
+from prompt_toolkit.shortcuts import confirm
 from rich.table import Table
 from rich.text import Text
 from rich.rule import Rule
 
+from goose.utils.execute_shell import execute_shell
+from goose.utils.path_file_output import show_diff
+from goose.utils.safe_mode import is_in_safe_mode
+
 RULESTYLE = "bold"
 RULEPREFIX = f"[{RULESTYLE}]───[/] "
 
+TASKS_WITH_EMOJI = {"planned": "⏳", "complete": "✅", "failed": "❌", "in-progress": "🕑", "cancelled": "🚫", "skipped": "⏩"}
```

Comment:
:nit: could we add a trailing comma so this `dict` prints out each line so it's easier to read?